pkg/capture/crd_to_job.go

```diff
@@ -519,9 +520,30 @@ func (translator *CaptureToPodTranslator) validateTargetSelector(captureTarget r
 	if captureTarget.NodeSelector == nil && captureTarget.PodSelector == nil {
 		return fmt.Errorf("Neither NodeSelector nor NamespaceSelector&PodSelector is set.")
 	}
```

Comment:
Out of curousity, What if someone goes ahead and sets namespaceSelector, but not set the podSelector, do we just ignore it in that case?

Reply:
If namespace selector is defined, but pod selector one of two things will happen.

if NodeSelector is nil
The if statement you highlighted will be hit and an error is thrown

if NodeSelector is not nil
The next if statement will be hit and an error is thrown (that NodeSelector and Namespace&PodSelectors are incompatible)

Reply:
> the logic I decided on will check whether NodeSelector is the default value, if yes then it will override it to use Pod&Namespace, if its an explicit value (aka not default) an error is thrown.

I am open to feedback though if folks think that NodeSelector shouldn't be the default if no parameters are provided.